src/services/lsp-factory.service.ts

```diff
@@ -1,9 +1,10 @@
 import { NonceManager } from "@ethersproject/experimental";
-import { LSPFactory } from "@lukso/lsp-factory.js";
+import { LSPFactory } from "@lukso/lsp-factory.js-alpha";
 import { getSigner } from "./provider.service";
 
 export async function getLspFactory(): Promise<LSPFactory> {
   const { provider, signer } = await getSigner();
   const nonceManager = new NonceManager(signer);
+  //@ts-ignore
```

Comment:
This is still needed?

Reply:
I guess not

Reply:
Let's remove then

Reply:
Let's remove then

Reply:
Still waiting for Andrew's PR to get merged, removing it might cause issues